### PR TITLE
Recipe for inspector-apm/inspector-symfony

### PR DESCRIPTION
**Pull request by ilvalerione**
| Q             | A
| ------------- | ---
| License       | MIT

The recipe is for the inspector package: [https://github.com/inspector-apm/inspector-symfony](https://github.com/inspector-apm/inspector-symfony)

### CODE REVIEW

File: inspector-apm/inspector-symfony/1.0/config/packages/inspector.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ingestion_key: '%env(INSPECTOR_INGESTION_KEY)%'
+    url: https://ingest.inspector.dev
+    enabled: true
+    unhandled_exceptions: true
+    query: true
+    query_bindings: false
+    templates: true
+    user: true
+    transport: async
+    server_sampling_ratio: 0
```
Comment (Nyholm) [inline inspector-apm/inspector-symfony/1.0/config/packages/inspector.yaml:11]: Do we really need all these default values? 

I would prefer a link to the bundles documentation instead.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. Just remove the default documentation and I'll be happy.

File: inspector-apm/inspector-symfony/1.0/config/packages/inspector.yaml (hunk 1 above)
Comment (ilvalerione) [reply inspector-apm/inspector-symfony/1.0/config/packages/inspector.yaml]: Hi, I removed the not mandatory config parameters.

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**